target/linux/ipq40xx/image/generic.mk

```diff
@@ -636,6 +636,19 @@ define Device/netgear_ex6150v2
 endef
 TARGET_DEVICES += netgear_ex6150v2
 
+define Device/netgear_wac510
```

Comment:
The commit message mentions the process of building that "openwrt_firmware.tar".
To me, it looks like this could be integrated further into a dedicated `Build/netgear-tar`makefile script
that packs the ubi into a tar archive together with the checksum and version file.
Or is this sadly a pipe-dream and there's something that I didn't account?

Reply:
Yeah, I had to look into how all this works lacking any prior such knowledge. It is still all rather a little bit hard-coded but, hey, it works! Feedback welcome.